src/setupProxy.js

```diff
@@ -0,0 +1,9 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
```

Comment:
is this required to be committed ?

Reply:
I think it should be added as it is part of project, though it is required only when we run dashboard locally.
It was in .gitignore previously.

Reply:
got it. 
ideally we should pick value of these. variables from env so in future any developer can change it for local development without requiring git commit.

```        
        target: 'http://demo.devtron.info:32080',
        changeOrigin: true,
        logLevel: 'info'
```